make_sukap_ROOT5.sh

```diff
@@ -16,7 +16,7 @@ if [ ! -d ${build_directory} ]; then
 	if [ -d ${G4WORKDIR} ]; then
 		rm -r ${G4WORKDIR}
 	fi
-	rm *.o *.a *.so *~ */*~ src/*Dict*
```

Comment:
Here

Reply:
WCSimRootDict was moved from `./src/` to `./`, but for some users, WCSimRootDict is still in `./src/`.
The modification here, and in the other scripts and GNUmakefile, removes the deletion of the WCSimRootDict in `./src/`. 

So, if a user still have  WCSimRootDict is still in `./src/` and update WCSim without cleaning it first, it will failed to compile, and will need to manually remove WCSimRootDict in  `./src/`. I wanted to keep this removal for some times, in order to give time to user to clean their directories automatically after the update. But @JacekHoleczek is against this, and points that we should not removed files which are not created during the compilation (I agree with him on this point, but I think cleaning old files is an acceptable exception to this rule).

Anyway, let's go with the current version, and I will make another PR to merge the compilation scripts for cmake and add a script to clean up old files like src/WCSimRootDict*